functionnal_tests/cypress/integration/login/login_after_disconnection.js

```diff
@@ -11,6 +11,8 @@ describe('Login after a disconnection ', function () {
   before(() => {
     cy.resetDB()
     cy.setupBaseDB()
+    cy.visitPage({ pageName: p.LOGIN })
+    cy.changeLanguage('en')
```

Comment:
Does this work if Tracim is already in English? I'm expecting this may fail in the CI.

We may need to force Cypress to run Chrome in English (or such that it opens contents in English so everybody has the same configuration.

Reply:
I don't know why, but every time I reloaded the page, it was in french, so I kind of forced it to be in English. Could this be because of my default settings or something like that ? 
I kind of had to "force" it to be in English on this page.

Reply:
I have the same issue, I don't know how to fix this, I haven't looked into it. We may have to look it up.

Reply:
I just checked, and yes, it works if Tracim is already in English

Reply:
I just looked it up with marie and, when logging out, since there is no user connected. So the frontend has to setup a language and it does so by using the browser language which can be different depending on computer running the test.
We can assume the ci is in english by default but developers might not be

Reply:
Note that this can also happen when creating a user inside the cypress test if you do not explicitly set the language